### PR TITLE
Minor Fixes on Rad and Gax

### PR DESCRIPTION
**Pull request by Ichaie**
# Description

minor on the maps rad and gax

---

# TODO

---

<details><summary><h1>Media</h1></summary>
<p>

</p>
</details>

---

# Changelog

:cl:
- fix: minor fixes on the maps Rad and Gax, fix te uranium crarte to filled

### CODE REVIEW

**Comment by VMSolidus** [comment]
Ooh what's being fixed?

**Comment by Ichaie** [comment]
I accidentally removed an atmos door and put in 2 new lockers, a bartender and a janitor.

File: Resources/Maps/gaxstation.yml [hunk 1]
```diff
@@ -128949,6 +129717,17 @@ entities:
...
+  entities:
+  - uid: 20352
+    components:
+    - type: Transform
+      parent: 20345
+    - type: Physics
+      angularDamping: 0
+      linearDamping: 0
+      canCollide: False
+    - type: InsideEntityStorage
```
Comment (VMSolidus) [inline Resources/Maps/gaxstation.yml:129730]:
```suggestion
```
Actually this should not be mapped. The Bartender gets their shotgun from their loadouts. They also have weapon options, in the form of optionally starting with a Mosin(rubber) in place of a shotgun.

**Review by VMSolidus — CHANGES_REQUESTED** [review]

File: Resources/Maps/gaxstation.yml (hunk 1 above)
Comment (Ichaie) [reply Resources/Maps/gaxstation.yml:129730]: Oh, I didn't realize that, sorry

**Comment by VMSolidus** [comment]
image

Test fail is valid, you'll need to ctrl-F the offending Uid and delete it from wherever it is(It was probably the Shotgun entity)

**Comment by VMSolidus** [comment]
Wuh? It's a super easy fix, I can do it right now.

**Review by VMSolidus — APPROVED** [review]

**Review by Peptide90 — APPROVED** [review]

**Pull request closed without merging**